controlpanel/api/slack.py

```diff
@@ -0,0 +1,20 @@
+from django.conf import settings
+import slack
+
+
+CREATE_SUPERUSER_MESSAGE = "`{username}` was created as a superuser"
+GRANT_SUPERUSER_ACCESS_MESSAGE = "`{username}` was granted superuser status"
+
+
+def notify_team(message, request_user=None):
```

Comment:
This function doesn't need to depend on the whole thing, it just needs a username.

To be honest I preferred this before when it was simpler and it was just getting a string (as it's potentially more reusable).

Reply:
I agree, I think I changed it to pass the user for some intermediate change to do with the other calling places that were then removed.